go/vt/vttablet/tabletmanager/vreplication/external_connector.go

```diff
@@ -131,8 +132,9 @@ func (c *mysqlConnector) Close(ctx context.Context) error {
 	return nil
 }
 
-func (c *mysqlConnector) VStream(ctx context.Context, startPos string, tablePKs []*binlogdatapb.TableLastPK, filter *binlogdatapb.Filter, send func([]*binlogdatapb.VEvent) error) error {
-	return c.vstreamer.Stream(ctx, startPos, tablePKs, filter, throttlerapp.ExternalConnectorName, send)
+func (c *mysqlConnector) VStream(ctx context.Context, startPos string, tablePKs []*binlogdatapb.TableLastPK,
+	filter *binlogdatapb.Filter, send func([]*binlogdatapb.VEvent) error, options *binlogdatapb.VStreamOptions) error {
+	return c.vstreamer.Stream(ctx, startPos, tablePKs, filter, throttlerapp.ExternalConnectorName, send, nil)
```

Comment:
I assume the reason we don't pass in options here is that we are streaming directly from mysql?
It's difficult to understand why each of these connectors exists and how they are used. We need some documentation at the struct level for each of these describing why.

Reply:
Yes, the options are not relevant here. We should see about refactoring those connectors and probably merge them into a single one, with config options. They share a lot of the same code.